src/applications/vaos/utils/appointment.jsx

```diff
@@ -1,10 +1,13 @@
-import React from 'react';
 import moment from './moment-tz';
 import guid from 'simple-guid';
 import environment from 'platform/utilities/environment';
-import { APPOINTMENT_TYPES, TIME_TEXT, PURPOSE_TEXT } from './constants';
-import FacilityAddress from '../components/FacilityAddress';
-import FacilityDirectionsLink from '../components/FacilityDirectionsLink';
+import {
```

Comment:
Should we rename this file to `appointment.js` since there's no longer any JSX in it?